specification/quantum/data-plane/Microsoft.Quantum/preview/2021-05-06-preview/quantum.json

```diff
@@ -0,0 +1,757 @@
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Target id."
+        },
+        "currentAvailability": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Target availability.",
+          "enum": [
+            "Available",
+            "Degraded",
+            "Unavailable"
+          ],
+          "x-ms-enum": {
+            "name": "TargetAvailability",
+            "modelAsString": true
+          }
+        },
+        "averageQueueTime": {
+          "type": "integer",
+          "format": "int64",
+          "readOnly": true,
+          "description": "Average queue time in seconds."
+        },
+        "statusPage": {
+          "type": "string",
+          "readOnly": true,
+          "description": "A page with detailed status of the provider."
+        }
+      }
+    },
+    "QuotaList": {
+      "description": "List of quotas.",
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Quota"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Link to the next page of results."
+        }
+      }
+    },
+    "Quota": {
+      "description": "Quota information.",
+      "type": "object",
+      "properties": {
+        "dimension": {
+          "type": "string",
+          "description": "The name of the dimension associated with the quota."
+        },
+        "scope": {
+          "type": "string",
+          "description": "The scope at which the quota is applied.",
+          "enum": [
+            "Workspace",
+            "Subscription"
+          ],
+          "x-ms-enum": {
+            "name": "DimensionScope",
+            "modelAsString": true
+          }
+        },
+        "providerId": {
+          "type": "string",
+          "description": "The unique identifier for the provider."
+        },
+        "utilization": {
+          "type": "number",
+          "description": "The amount of the usage that has been applied for the current period."
+        },
+        "holds": {
+          "type": "number",
+          "description": "The amount of the usage that has been reserved but not applied for the current period."
+        },
+        "limit": {
+          "type": "number",
+          "description": "The maximum amount of usage allowed for the current period."
+        },
+        "period": {
+          "type": "string",
+          "description": "The time period in which the quota's underlying meter is accumulated. Based on calendar year. 'None' is used for concurrent quotas.",
+          "enum": [
+            "None",
+            "Monthly"
+          ],
+          "x-ms-enum": {
+            "name": "MeterPeriod",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "restError": {
```

Comment:
The model name should be camelcase

Reply:
Do you mean pascal case (RestError) - I'll change it to be consistent.